system-int-test/test/cmr/system_int_test/search/subscription/subscription_search_test.clj

```diff
@@ -527,3 +527,29 @@
       "Sort by name then provider id descending order"
       ["-name" "-provider"]
       [subscription2 subscription1 subscription4 subscription3])))
+
+(deftest json-result-format-test
```

Comment:
We could delete this test and fold the key checking into the `cmr.system-int-test.utils.subscription-util/search-json` function by modifying `search/process-response`.

Reply:
process-response specifically modifies the response to use kebab-cases for all map keys. Having that be aware of what type of query makes less sense than comparing an unprocessed response available.